Source/WebKit/Platform/IPC/HandleMessage.h

```diff
@@ -109,10 +110,10 @@ struct CodingType<std::tuple<Ts...>> {
 
 template<typename T, typename C, typename MF>
 void handleMessage(Decoder& decoder, C* object, MF function)
-{
+{notImplemented();
```

Comment:
now you are using notImplemented in functions that are in fact implemented?
Wouldn't it make more sense to use BeDC at a lower level (in the logging framework of webkit) and use LogWithStream or whatever log support WebKit offers?